code/mobs/_mob.dm

```diff
@@ -4,6 +4,7 @@
 	layer = MOB_LAYER
 	see_invisible = SEE_INVISIBLE_LIVING
 	shadow_size = 2
+	sight = SEE_SELF|SEE_PIXELS
```

Comment:
Because there are a tremendous number of visual issues without it. You can see objects with a single pixel offset over a wall boundary, doors extend into the view occlusion, lighting becomes strange at time. Why do we want to avoid it?